static/authui/css/main.css

```diff
@@ -105,7 +105,7 @@ h1, h2, h3, h4, h5, h6 {
   border: none;
   border-radius: 0;
   /* NOTE(1) */
-  font-size: 16px;
+  font-size: 1rem;
```

Comment:
We didn't set font size at root element right? Since we don't import normalizer css, let's set it to 16px explicitly at root element.

Reply:
I believe the default font size should be most accessible to the user. Setting it to 16px is not accessible. The intention here is to use whatever default font size the user agent has set for the user.